examples/hikari_lightbulb_basic_queue/music_plugin.py

```diff
@@ -37,51 +38,45 @@ async def track_exception(self, lavalink, event):
 
```

Comment:
i can't type-hint the eventhandler as the types should prolly come from lavasnek (same goes for the pure hikari example)

Reply:
There's no way to typehint a function call. The function itself is not defined on lavasnek, it just looks with a function with that name on the class instance you give to the LavalinkBuilder.

Reply:
I mean the arguments...

Reply:
This is how the event functions are called on Rust, that function call doesn't have, nor it checks for typing.
```rust
fn call_event<T: Send + Sync + pyo3::IntoPy<PyObject> + 'static>(
    handler: &LavalinkEventHandler,
    client: LavalinkClient,
    event: T,
    name: &'static str,
) {
    let slf1 = handler.clone();
    let slf2 = handler.clone();

    Python::with_gil(|py| {
        // Get the current async event loop
        let current_loop = slf1.current_loop.as_ref(py);

        pyo3_asyncio::tokio::future_into_py_with_loop(current_loop, async move {
            let future = Python::with_gil(|py| {
                // This gets the EventHandler class provided to the Builder.
                let py_event_handler = slf2.inner.as_ref(py);
                // Call the method with the name of the event, passing the client and the event data as parameters.
                let coro_result =
                    py_event_handler.call_method(name, (Lavalink { lava: client }, event), None);

                // Convert the coro into a rust Future
                if let Ok(coro) = coro_result {
                    pyo3_asyncio::tokio::into_future(coro)
                } else {
                    Err(error::NameError::new_err("Undefined event"))
                }
            });

            // Await the Future
            if let Ok(f) = future {
                f.await.unwrap();
            }

            Ok(Python::with_gil(|py| py.None()))
        })
        .unwrap();
    });
}
```

Reply:
I mean that I have to wait for stub files until I can type-hint the client and event so that for example your code completion knows what attrs they have

Reply:
Or are they exposed to python?

Reply:
```py
from lavasnek_rs import *

class EventHandler:
    async def stats(self, lava_client: Lavalink, event: Stats):
        print(event)
    async def player_update(self, lava_client: Lavalink, event: PlayerUpdate):
        print(event)
    async def track_start(self, lava_client: Lavalink, event: TrackStart):
        print(event)
    async def track_exception(self, lava_client: Lavalink, event: TrackException):
        print(event)
    async def track_finish(self, lava_client: Lavalink, event: TrackFinish):
        print(event)
    async def websocket_closed(self, lava_client: Lavalink, event: WebSocketClosed):
        print(event)
    async def player_destroyed(self, lava_client: Lavalink, event: PlayerDestroyed):
        print(event)
```